README.md

```diff
@@ -1,31 +1,64 @@
-# flottbot
+<p align="center">
+  <img alt="flottbot logo" src="https://i.imgur.com/P9NI00w.png" height="160" />
+
+  <h3 align="center">Flottbot</h3>
+</p>
+
+--------
 
 [![GoDoc](https://godoc.org/github.com/target/flottbot?status.svg)](https://godoc.org/github.com/target/flottbot)
 [![Build Status](https://travis-ci.org/target/flottbot.svg)](https://travis-ci.org/target/flottbot)
+[![GitHub release](https://img.shields.io/github/release/target/flottbot.svg)](https://github.com/target/flottbot/releases/latest)
 [![Coverage Status](https://coveralls.io/repos/target/flottbot/badge.svg?branch=master)](https://coveralls.io/r/target/flottbot?branch=master)
 [![Go Report Card](https://goreportcard.com/badge/github.com/target/flottbot)](https://goreportcard.com/report/github.com/target/flottbot)
+[![docker](https://img.shields.io/docker/automated/target/flottbot.svg)](https://hub.docker.com/r/target/flottbot)
 
-A chatbot framework written in Go. But there's a catch, you don't need to know a lick of Go! Configure your bot via YAML files, extend functionality by writing scripts in your favorite language.
+Flottbot is a chatbot framework written in Go. But there's a catch, you don't need to know a lick of Go! Configure your bot via YAML files, extend functionality by writing scripts in your favorite language.
 
 The philosophy behind flottbot is to create very simple, lightweight, "dumb" bots that interact with APIs and scripts which house a bot's business logic. The word **flott** comes from the German word meaning _quick_/_speedy_.
 
+1. [Installation](#installation)
+1. [Docker images](#docker-images)
+1. [Available providers](#available-providers)
```

Comment:
I feel like providers may not be the best wording for this section.... Any thoughts?

Reply:
Maybe stick to the codebase terminology of `remotes`?